helpers/config.js

```diff
@@ -171,6 +171,14 @@ function Config(packageJson, parseCommandLineOptions = true) {
 
 		validateForce(appConfig);
 
+		// Disable api, peers, broadcasts, syncing and ws workers when snapshotting mode required
+		if (program.snapshot) {
+			appConfig.api.enabled = false;
+			appConfig.peers.enabled = false;
+			appConfig.peers.list = [];
+			appConfig.broadcasts.active = false;
+			appConfig.syncing.active = false;
+		}
```

Comment:
I don't like overriding config attributes without telling user in the logs that some of his config attributes are override. Consider the fact user is using config some pm2 config file and actually not aware of it the flags in there. He can't see his API working, he came and check the config file and will see its there and set to true. Then he will get confused why API is not working.